digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/csvDataImporter/MassTransitStopCsvImporter.scala

```diff
@@ -42,9 +44,17 @@ class MassTransitStopCsvOperation(vvhClientImpl: VVHClient, roadLinkServiceImpl:
       r => r.toList.map ( p => (p._1.toLowerCase, p._2)).toMap
     }
 
-    val strategy = getStrategy(csvRow.head)
-    strategy.importAssets(csvRow: List[Map[String, String]], fileName, user, logId, roadTypeLimitations)
-    }
+    try {
+      val strategy = getStrategy(csvRow.head)
+      strategy.importAssets(csvRow: List[Map[String, String]], fileName, user, logId, roadTypeLimitations)
+
+    } catch {
+      case e: Exception =>
+        OracleDatabase.withDynTransaction  {
+        importLogDao.update(logId, Status.Abend, Some("Latauksessa tapahtui odottamaton virhe: " + e.toString))
+        }
+      }
+  }
```

Comment:
This try catch is not being done in line 282?

Reply:
This is necessary because when the strategy fails it's being generated a throw exception not caught and the error is not logged.